packages/components/pie-webc-core/src/test/functions/dispatchCustomElement/dispatchCustomEvent.browser.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { test, expect } from '@sand4rt/experimental-ct-web';
+import { MockComponent } from './MockComponent.ts';
+
+test.describe('dispatchCustomEvent function', () => {
+    test('should dispatch an event when called', async ({ mount, page }) => {
+        // Arrange
+        const events : Array<Event> = [];
+
+        await mount(MockComponent, {
+            on: {
+                'pie-mock-event': (event: Event) => events.push(event),
+            },
+        });
+
+        // Act
+        await page.locator('[data-test-id="dispatch-custom-event-mock"]').click();
```

Comment:
```suggestion
        await component.click();
```

Reply:
please can you give these two commits a try?

Reply:
Interestingly only `await component.locator('[data-test-id="dispatch-custom-event-mock"]').click();` works. I compared the code with pie-button and don't see why there `await component.click();` does the trick 🤔